src/aws4_request.gleam

```diff
@@ -9,20 +9,57 @@ import gleam/string
 
 // TODO: document
 // TODO: document params
-// https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html
-// https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-header-based-auth.html
-pub fn sign(
+// https://docs.aws.amazon.com/IAM/latest/UserGuide/create-signed-request.html
```

Comment:
I prefer this link because it's service agnostic (the same v4 signing works for all AWS services) and it appears to be more detailed in describing the implementation than the single service links.